pbiviz.json

```diff
@@ -4,12 +4,12 @@
     "displayName": "barChart",
     "guid": "PBI_CV_9894B302_1DFF_4A96_ABFE_BF8588197166",
     "visualClassName": "BarChart",
-    "version": "4.0.0",
+    "version": "4.0.0.0",
```

Comment:
why not 4.1.0 or 6.0.0?

Reply:
yeah it should be 4.1.0.0,  the four digits is needed in when using the newest pbi-visual-tools